# These are the valid object types that may be specified for an object.
string STANDARD=standard
string QRC=qrc
string OFFAXIS=off_axis
string EMERGENT=emergent

# Object type.
# Must be one of the constants defined above.
string data
